source/jormungandr/jormungandr/instance.py

```diff
@@ -333,6 +343,14 @@ def get_realtime_proxies_from_db(self):
         result = models.external_services if models else None
         return [res for res in result if res.navitia_service == 'realtime_proxies']
 
+    def get_bss_stations_from_db(self):
```

Comment:
minor:
```suggestion
    def get_bss_stations_services_from_db(self):
```
or provider if it suits you more